__tests__/app.test.js

```diff
@@ -191,9 +191,71 @@ describe("POST /api/reviews/:review_id/comments", () => {
       .send(commentObj)
       .expect(400)
       .then((response) => {
-        expect(response.body.message).toBe(
-          "post object contains invalid values"
-        );
+        expect(response.body.message).toBe("object contains invalid values");
+      });
+  });
+});
+
+describe("PATCH /api/reviews/:review_id", () => {
+  it("should return status 201 and respond with the updated review object", () => {
+    const votesUpdateObj = { inc_votes: 10 };
+    const expectedResponse = {
+      owner: expect.any(String),
+      title: expect.any(String),
+      review_id: expect.any(Number),
+      category: expect.any(String),
+      review_img_url: expect.any(String),
+      created_at: expect.any(String),
+      votes: expect.any(Number),
+      designer: expect.any(String),
+      review_body: expect.any(String),
+    };
```

Comment:
You can definitely be more specific here, in particular the votes value you want to check that the votes have increased to the number you expect!

If you are patching a specific review, instead of expect.any, use the values that make this test specific and more strict. Review_id should be 1 as well for example!